example_data/detect_variants/detect_variants.yaml

```diff
@@ -19,3 +19,9 @@ tumor_bam:
   class: File
   path: H_NJ-HCC1395-HCC1395_exome.bam
 strelka_exome_mode: true
+vep_cache_dir:
+  class: File
+  path: /gscmnt/gc2764/cad/ssiebert/toil_test/inputs/vep_cache_location.txt
+synonyms_file:
+  class: File
+  path: /gscmnt/gc2764/cad/ssiebert/toil_test/inputs/chromAlias.ensembl.txt
```

Comment:
Thanks for the pointer. I updated the yaml. I also copied my vep_cache_location.txt into this location and amended the path accordingly. The txt still points to the actual path in my location but I don't think there is a good way around it since we don't want to make the cache files part of this repo.